staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -328,6 +333,15 @@ type ListOptions struct {
 	// Timeout for the list/watch call.
 	// +optional
 	TimeoutSeconds *int64 `json:"timeoutSeconds,omitempty" protobuf:"varint,5,opt,name=timeoutSeconds"`
+
+	// From is the continuation of a previous list call and will begin the list at the appropriate
+	// location. It is an qpaque value and may change serialization across server versions. If the
+	// continuation is not possible an error will be returned and you must start over. This may
+	// return an empty response in the event all requested fields are filtered out.
+	From string `json:"from,omitempty" protobuf:"bytes,7,opt,name=from"`
+	// Limit is a maximum number of responses to return for a list call. If more items exist, the
+	// server will set the next field on the list metadata to a token you can pass as from.
+	Limit int64 `json:"limit,omitempty" protobuf:"varint,8,opt,name=limit"`
```

Comment:
+1 - there definitely should be a different between those two

@smarterclayton Regarding `*int64` support - I think we do support it. E.g. "TimeoutSeconds" 10 lintes above is actually of time `*int64`.